mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -120,6 +126,16 @@ class _ChatResponse(pydantic.BaseModel, extra="forbid"):
     usage: _ChatUsage
 
 
+class _ChatChunkResponse(pydantic.BaseModel, extra="forbid"):
```

Comment:
Do we also need CompletionsChunk?

Reply:
Currently, the completion model `predict` returns string, so that to keep consistency, I prefer to also make `predict_stream` returns an iterator of string.

Reply:
Another reason to do not make conversion to CompletionsChunk for langchain:

Langchain has various kinds of model or pipeline, judging whether it is a completion model is tricky.

Reply:
@serena-ruan thoughts ?